src/color-chart/README.md

```diff
@@ -24,4 +24,54 @@ By default, the other coordinate would be the index of the color in the list, bu
 <color-chart y="oklch.l">
 	<color-scale colors="50: #e3f2fd, 100: #bbdefb, 200: #90caf9, 300: #64b5f6, 400: #42a5f5, 500: #2196f3, 600: #1e88e5, 700: #1976d2, 800: #1565c0, 900: #0d47a1"></color-scale>
 </color-chart>
+```
+
+You can optionally specify a label for a data point by  dividing it from the other coordinate with `/` (slash):
+
+```html
+<color-chart y="oklch.l">
+	<color-scale colors="
+		Blue 50 / 50: #e3f2fd,
```

Comment:
This repetition should not be needed (and I already have code for that — though no idea if it works)